src/main/java/org/prebid/server/exception/RejectedRequestException.java

```diff
@@ -0,0 +1,16 @@
+package org.prebid.server.exception;
+
+import org.prebid.server.hooks.execution.model.HookExecutionContext;
+
+public class RejectedRequestException extends RuntimeException {
+
+    private final HookExecutionContext hookExecutionContext;
```

Comment:
Why we need it here?

Reply:
In order to get debug information from context and put it in response.